### DISCUSSION TITLE
Reference list of wake model parameters

### DISCUSSION THREAD

**Discussion by tupaibasu2004** [NatLabRockies/floris#469] [Q&A] [answered]
Hi,

Can anybody suggest where can I find a comprehensive list of parameter values that can be passed to floris .json file? For example, consider the entry "deflection_model": "gauss". Apart from "gauss", which other values can be assigned to the key "deflection_model". SImilarly, for other key values of .json file, which all are the possible options and where can I find those?

**Comment by rafmudaf** [comment, marked as answer]
Hi @tupaibasu2004 we still do not have a reference list of these parameters. However, we are working on restructuring and improving the online documentation, so I've made a note to include this information there. See https://github.com/NREL/floris/discussions/468 and please let me know if you have anything more to add on this topic.

**Reply by tupaibasu2004** [reply]
Thanks Rafael.
